api.planx.uk/admin/index.html

```diff
@@ -0,0 +1,119 @@
+      margin: auto;
+      margin-top: 2em;
+    }
+
+    table {
+      width: 100%;
+      border-collapse: collapse;
+      border-width: 2px;
+      border-color: #0010a4;
+      border-style: none;
+      color: #000000;
+      font-size: 1em;
+    }
+
+    table td,
+    table th {
+      border-width: 2px;
+      border-color: #0010a4;
+      border-style: solid;
+      padding: 10px 5px;
+    }
+
+    table thead {
+      background-color: #0010a4;
+      color: #fff;
+      text-align: left;
+    }
+  </style>
+</head>
+<div>
+  <h1>PlanX <code>/admin</code> endpoints</h1>
+  <p>
+    Admin endpoints are available to developers and editors for viewing,
+    downloading, and troubleshooting data about Planx services and sessions.
+  </p>
+  <p>
+    These endpoints are only accessible to users who are signed into the Editor,
+    have a white-listed email address, and are connecting via a WARP client.
+  </p>
+  <table>
+    <thead>
+      <tr>
+        <th>Endpoint</th>
+        <th>Optional params</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>/feedback</code></td>
+        <td></td>
+        <td>Downloads a FeedbackFish CSV</td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/summary</code></td>
+        <td></td>
+        <td>
+          Returns a passport, breadcrumbs, and other key details for a given
+          session
+        </td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/xml</code></td>
+        <td></td>
+        <td>
+          Generates a OneApp XML, relies on <code>uniform_applications</code>
+        </td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/bops</code></td>
+        <td></td>
+        <td>
+          Generates a BOPS payload, relies on <code>bops_applications</code>
+        </td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/csv</code></td>
+        <td><code>?download=true</code></td>
+        <td>Generates or downloads an application CSV file</td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/csv-redacted</code></td>
+        <td><code>?download=true</code></td>
+        <td>
+          Generates or downloads an application CSV file with personal details
+          redacted
+        </td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/html</code></td>
+        <td></td>
+        <td>Generates an application overview HTML</td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/html-redacted</code></td>
+        <td></td>
+        <td>
+          Generates an application overview HTML with personal details redacted
+        </td>
+      </tr>
+      <tr>
+        <td><code>/session/:sessionId/zip</code></td>
+        <td><code>?includeXML=true</code></td>
+        <td>
+          Generates a zip file for Send to Email, or Uniform when XML is
+          included
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
```

Comment:
html doesn't feel like a particularly clever solution, but looks okay and is easy enough to update? open to other suggestions though!